server/routes/search.ts

```diff
@@ -1,13 +1,13 @@
-import url from 'url'
 import { type RequestHandler, Router } from 'express'
 import { body } from 'express-validator'
-import { validatePrisonerSearch, validateVisitSearch } from './searchValidation'
-import config from '../config'
-import { getResultsPagingLinks } from '../utils/utils'
-import { VisitInformation } from '../@types/bapv'
+import url from 'url'
```

Comment:
Not sure if you need `import url from 'node:url'` ? https://nodejs.org/api/url.html#url

Reply:
just using what was already being used, also being used on another route `vistsReviewListingsController` - so made sense to keep the formatting the same. Happy to change over if you think it's necessary though

Reply:
No worries, I know they keep moving some things internally in node. I think that import does it locally from node anyway 🙂